gallery/visualtests/nxDropdown.js

```diff
@@ -90,6 +90,25 @@ describe('NxDropdown', function() {
 
   });
 
+  describe('NxDropdown with NxThreatIndicator', function() {
+    const selector = '#nx-dropdown-with-nx-threat-indicator-example .nx-dropdown';
+
+    beforeEach(async function() {
+      const [button] = await waitAndGetElements(selector + ' .nx-dropdown__toggle');
+
+      await button.click();
+    });
+
+    it('looks right', async function() {
+      const [targetElement] = await waitAndGetElements(selector);
+
+      await moveMouseAway();
+
+      await checkScreenshot(targetElement, 251, 252);
```

Comment:
It looks like a few px could/should be trimmed off of the bottom of this to minimize the risk of unrelated changes to the documentation page causing diffs.